model/post_list.go

```diff
@@ -9,8 +9,9 @@ import (
 )
 
 type PostList struct {
-	Order []string         `json:"order"`
-	Posts map[string]*Post `json:"posts"`
+	Order    []string         `json:"order"`
+	Posts    map[string]*Post `json:"posts"`
+	Profiles UserMap          `json:"profiles"`
```

Comment:
Awesome idea to include the user profiles with the posts returned from the server